Zend/Optimizer/zend_func_info.c

```diff
@@ -834,6 +834,72 @@ static const func_info_t func_infos[] = {
 static HashTable func_info;
 ZEND_API int zend_func_info_rid = -1;
 
+void zend_verify_internal_func_info(zend_function *fn, zval *retval) {
+	if (fn->common.scope) {
+		/* This is a method, not a function. */
+		return;
+	}
+
+	zend_string *name = fn->common.function_name;
+	if (!name) {
+		/* The pass function has no name. */
```

Comment:
I hadn't heard of that function.

```suggestion
		/* The pass function (zend_pass_function) has no name. */
```